modules/services/cloud-connector-ecs/ecs-service.tf

```diff
@@ -16,6 +16,7 @@ resource "aws_ecs_service" "service" {
   task_definition       = aws_ecs_task_definition.task_definition.arn
   wait_for_steady_state = true
   tags                  = var.tags
+  propagate_tags        = "SERVICE"
```

Comment:
How about ServiceFamily too?

Reply:
The AWS Terraform provider currently only supports propagation of tags from either the `SERVICE` or `TASK_DEFINITION`.